File: TypeScript/Operator/equals-operator.ts
function main(): void {
    // Check if two strings are the same and return a boolean
    console.log(checkTwoItemsAreEqual("Hello", "Hello"))
}

main()

// Check if two strings are the same and return a boolean
function checkTwoItemsAreEqual(primary: string, secondary: string): boolean {
    return primary == secondary
}

File: Go/Operator/equals-operator.go
package main

import (
	"fmt"
)

func main() {
	// Check if two values are the same and return a bool.
	fmt.Println(equalsOperator(1, 1))
}

// Check if two values are the same and return a bool.
func equalsOperator(primary int, secondary int) bool {
	return primary == secondary
}
